modern/src/settings/GeofencePage.jsx

```diff
@@ -76,7 +76,7 @@ const GeofencePage = () => {
                 label={t('sharedDescription')}
               />
               <SelectField
-                value={item.calendarId || 0}
+                value={item.calendarId || null}
```

Comment:
`|| null` is probably not needed here and in other places

Reply:
It'll probably generate warnings if item.calendarId (and similar) will be `false`. But I will check this.

Reply:
It would never be false though, right? It will either be not set or an id.

Reply:
Removing this actually leads to a MUI/React error:

> MUI: A component is changing the uncontrolled value state of Autocomplete to be controlled.
> Elements should not switch from uncontrolled to controlled (or vice versa).
> Decide between using a controlled or uncontrolled Autocomplete element for the lifetime of the component.
> The nature of the state is determined during the first render. It's considered controlled if the value is not `undefined`.
> More info: https://fb.me/react-controlled-components

Reply:
After the last commit we are now defaulting to null and it works as expected across the board.